mjs_firmware.ino

```diff
@@ -300,6 +310,9 @@ void getPosition()
 
   if (gps.statistics.ok == 0)
     Serial.println(F("No GPS data received, check wiring"));
+  
+  gpsSerial.flush();
```

Comment:
I think "flush()" only flushes pending writes. Since we only read from the serial port, I do not think this should be needed.

Reply:
@matthijskooijman Thanks for letting me know. In that case, I will revert this change.
```suggestion
```